Jenkinsfile

```diff
@@ -5,7 +5,9 @@ def pipeline = new org.js.AppPipeline(
     test:               false,
     dockerRegistryCred: 'bot-sora2-rw',
     dockerImageName:    'sora2/subquery',
-    buildDockerImage:   'docker.soramitsu.co.jp/build-tools/node:14-ubuntu',
-    preBuildCmds:       ['yarn install', 'yarn codegen']
+    npmRegistries:      [:],
```

Comment:
do you need this var?

Reply:
Yes, because in the npm image, npm-ci-login is installed, which is later launched. If you don't do this step, the build will fail.